docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -36,11 +36,10 @@ components:
       required: true
       schema:
         description: >
-          Either an 8-character alphanumeric unique identifier generated by the service
-          or an authority-provided identifier
+          Either a unique identifier generated by the service or an authority-provided identifier
         oneOf:
           - type: string
-            description: A short string: the group ID assigned by the service
+            description: A short string — the group ID assigned by the service
```

Comment:
nitpick: the restatement of it being a string feels kinda repetitive since the type indicates it's a string already.